appserver/security/core-ee/src/main/java/com/sun/enterprise/security/jmac/config/ConfigDomainParser.java

```diff
@@ -48,205 +53,171 @@
         defaultClientID = msgConfig.getDefaultClientProvider();
 
-        if (_logger.isLoggable(Level.FINE)) {
-            _logger.fine("Intercept Entry: " +
-                        "\n    intercept: " + intercept +
-                        "\n    defaultServerID: " + defaultServerID +
-                        "\n    defaultClientID:  " + defaultClientID);
+        if (_logger.isLoggable(FINE)) {
+            _logger.fine("Intercept Entry: " + "\n    intercept: " + intercept + "\n    defaultServerID: " + defaultServerID
+                    + "\n    defaultClientID:  " + defaultClientID);
         }
 
         if (defaultServerID != null || defaultClientID != null) {
             layersWithDefault.add(intercept);
         }
 
-        GFServerConfigProvider.InterceptEntry intEntry =
-            (GFServerConfigProvider.InterceptEntry)newConfig.get(intercept);
+        GFServerConfigProvider.InterceptEntry intEntry = newConfig.get(intercept);
 
         if (intEntry != null) {
-            throw new IOException("found multiple MessageSecurityConfig " +
-                                "entries with the same auth-layer");
+            throw new IOException("found multiple MessageSecurityConfig " + "entries with the same auth-layer");
         }
 
         // create new intercept entry
-        intEntry = new GFServerConfigProvider.InterceptEntry(defaultClientID,
-                defaultServerID, null);
+        intEntry = new GFServerConfigProvider.InterceptEntry(defaultClientID, defaultServerID, null);
         newConfig.put(intercept, intEntry);
         return intercept;
     }
 
-    private void parseIDEntry(ProviderConfig pConfig,
-			      Map newConfig, String intercept)
-	throws IOException {
+    private void parseIDEntry(ProviderConfig pConfig, Map<String, GFServerConfigProvider.InterceptEntry> newConfig, String intercept) throws IOException {
 
         String id = pConfig.getProviderId();
         String type = pConfig.getProviderType();
         String moduleClass = pConfig.getClassName();
-        MessagePolicy requestPolicy = 
-	    parsePolicy((RequestPolicy) pConfig.getRequestPolicy());
-        MessagePolicy responsePolicy = 
-	    parsePolicy((ResponsePolicy) pConfig.getResponsePolicy());
+        MessagePolicy requestPolicy = parsePolicy((RequestPolicy) pConfig.getRequestPolicy());
+        MessagePolicy responsePolicy = parsePolicy((ResponsePolicy) pConfig.getResponsePolicy());
 
         // get the module options
 
-        Map options = new HashMap();
-        String key;
-        String value;
+        Map<String, Object> options = new HashMap<>();
 
-	List<Property> pList = pConfig.getProperty();
+        List<Property> pList = pConfig.getProperty();
 
-	if (pList != null) {
+        if (pList != null) {
 
-	    Iterator<Property> pit = pList.iterator();
+            Iterator<Property> pit = pList.iterator();
 
-	    while (pit.hasNext()) {
+            while (pit.hasNext()) {
```

Comment:
This could be turned into a foreach loop

Reply:
Indeed, but this is a first pass clean. There's way to much files involved to fix every issue. This one mostly focussed on formatting, renaming and adding generics.